### PR TITLE
Add tap test jobs on CircleCI

### PR DESCRIPTION
**Pull request by taymoork2**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Test Coverage

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* SDK Version
* Node/Browser Version
* NPM Version

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -361,3 +390,32 @@ workflows:
...
+          requires:
+            - build
+      - tap:
+          package: '@webex/internal-plugin-conversation'
+          requires:
+            - build
+      - tap:
+          package: '@webex/internal-plugin-locus'
+          requires:
+            - build
```
Comment (InteractiveTimmy) [inline .circleci/config.yml]: Are these the only ones defined in the scope of the implementation requirements?

Comment (taymoork2) [reply .circleci/config.yml]: Nope, these are the ones the ones I remember (encryption & locus) or the ones we need (conversation)
I wanted you or Adam to provide more packages that directly touch services

Comment (adamweeks) [reply .circleci/config.yml]: `webex-core` and `plugin-meetings` would be the two big ones we want to test as well.

Comment (InteractiveTimmy) [reply .circleci/config.yml]: Update looks good! As discussed, our team should consider expanding these after a "trial" period of running just these packages.

**Pull request merged**